src/feature/flow_chart_panel/FlowChartTabView.tsx

```diff
@@ -98,7 +97,7 @@ const FlowChartTab = () => {
   const addNewNode = useAddNewNode(setNodes, getNodeFuncCount);
   const sidebarCustomContent = useMemo(
     () => <SidebarCustomContent onAddNode={addNewNode} />,
-    [nodes, edges]
+    []
```

Comment:
`addNewNode` is dependent on `nodes` and `edges` variables as it takes `getNodeFuncCount` as a parameter that needed to be updated each time `nodes` and `edges` get updated. So it's required to add them to the dependency list. Or we have to find another alternative for this.

Reply:
We can just observe `addNewNode` for changes, works perfectly.